Formula/czmq.rb

```diff
@@ -44,6 +44,10 @@ class Czmq < Formula
   depends_on "xmlto" => :build
   depends_on "zeromq"
 
+  on_ventura :or_newer do
+    depends_on "lz4"
```

Comment:
May want to use on everywhere or try to prevent linkage since will impact any build-from-source or rebuild/bumps.

Could be a side effect of `zstd` dependency on `lz4` - #106098 (as `zstd` is dependency of `asciidoc`).

```
`brew linkage --test czmq` failed on macOS Big Sur (11) on Apple Silicon!
Broken dependencies:
  /opt/homebrew/opt/lz4/lib/liblz4.1.dylib (lz4)
```